ortools/math_opt/solver_tests/logical_constraint_tests.cc

```diff
@@ -86,9 +86,22 @@ constexpr absl::string_view no_sos2_support_message =
 constexpr absl::string_view no_indicator_support_message =
     "This test is disabled as the solver does not support indicator "
     "constraints";
+constexpr absl::string_view no_updating_binary_variables_message =
+    "This test is disabled as the solver does not support updating "
+    "binary variables";
+constexpr absl::string_view no_deleting_indicator_variables_message =
+    "This test is disabled as the solver does not support deleting "
+    "indicator variables";
+constexpr absl::string_view no_incremental_add_and_deletes_message =
+    "This test is disabled as the solver does not support incremental "
+    "add/delete";
 
 // We test SOS1 constraints with both explicit weights and default weights.
 TEST_P(SimpleLogicalConstraintTest, CanBuildSos1Model) {
+  if (GetParam().solver_type == SolverType::kXpress) {
```

Comment:
I would suggest adding test suite parameters for these skipped tests (like supports_integer_variables, supports_indicator_constraints, etc).  
maybe we can get feedback from someone on the OR-Tools team on this.

Reply:
Added parameter in commit 35feb4ddecaa95b8a9cb9629c5e5580a38c288d6